src/dotnet-scaffolding/Microsoft.DotNet.Scaffolding.Core/Scaffolders/Scaffolder.cs

```diff
@@ -9,22 +9,22 @@ public class Scaffolder : IScaffolder
 {
     private readonly string _name;
     private readonly string _displayName;
-    private readonly string _category;
+    private readonly List<string> _categories;
     private readonly string? _description;
     private readonly List<ScaffolderOption> _options;
     private readonly List<ScaffoldStep> _steps;
     private readonly List<ScaffoldStepPreparer> _preparers;
     public string Name => _name;
     public string DisplayName => _displayName;
-    public string Category => _category;
+    public List<string> Categories => _categories;
```

Comment:
For a public API I would generally prefer this to be IEnumerable or ReadOnlyCollection or something like that rather than List (see [this](https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/guidelines-for-collections), though its been a while since I read it)

Reply:
yup no List<T> is public APIs, will switch.